remake_sources.yml

```diff
@@ -0,0 +1,3 @@
+LoLinR:
```

Comment:
This file tells remake where to find github packages, so that they are installed using `remake::install_missing_packages()`